ccc/oci.py

```diff
@@ -72,14 +73,19 @@ class _OciRequestHandler(logging.Handler):
     def __init__(
         self,
         level,
-        es_client,
+        es_client: ccc.elasticsearch.ElasticSearchClient,
```

Comment:
since it is very unlikely that the checked condition will change, rather than adding an early-exit, I would deem it more reasonable to simply not register this request-handler in the first place.